src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,108 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ]  Linters help maintain code quality by enforcing best practices and catching potential issues before they become problems. Disabling them can lead to lower code quality and increase the risk of introducing bugs. it is best to avoid disabling linters unless absolutely necessary. Instead, it is better to address any issues raised by the linters in the code.
- [ ] So I will suggest you remove this from your code :wink: